printers/prusa-mk3s/tmc2130-stealth.cfg

```diff
@@ -0,0 +1,78 @@
+# WARNING. DO NOT EDIT THIS FILE.
+# To override settings from this file, you can copy and paste the relevant
+# sections into your printer.cfg and change it there.
+
+[tmc2130 stepper_x]
+cs_pin: x_cs_pin
+interpolate: True
+hold_current: .25
+diag1_pin: !x_diag_pin
+driver_IHOLDDELAY: 8
+driver_TPOWERDOWN: 0
+driver_TBL: 2
+driver_TOFF: 3
+driver_HEND: 1
+driver_HSTRT: 5
+driver_PWM_FREQ: 2
+driver_PWM_GRAD: 2
+driver_PWM_AMPL: 230
+driver_PWM_AUTOSCALE: True
+driver_SGT: 5
+stealthchop_threshold: 1
+
+[tmc2130 stepper_y]
+cs_pin: y_cs_pin
+interpolate: True
+hold_current: .25
+diag1_pin: !y_diag_pin
+driver_IHOLDDELAY: 8
+driver_TPOWERDOWN: 0
+driver_TBL: 2
+driver_TOFF: 3
+driver_HEND: 1
+driver_HSTRT: 5
+driver_PWM_FREQ: 2
+driver_PWM_GRAD: 2
+driver_PWM_AMPL: 235
+driver_PWM_AUTOSCALE: True
+driver_SGT: 5
+stealthchop_threshold: 1
+
+
+[tmc2130 stepper_z]
+cs_pin: z_cs_pin
+interpolate: True
+hold_current: .35
+diag1_pin: !z_diag_pin
+driver_IHOLDDELAY: 8
+driver_TPOWERDOWN: 0
+driver_TBL: 2
+driver_TOFF: 3
+driver_HEND: 1
+driver_HSTRT: 5
+driver_PWM_FREQ: 2
+driver_PWM_GRAD: 4
+driver_PWM_AMPL: 200
+driver_PWM_AUTOSCALE: True
+driver_SGT: 4
+stealthchop_threshold: 1
+
+[tmc2130 extruder]
+cs_pin: e_cs_pin
+interpolate: True
+run_current: .52
+hold_current: .2
+sense_resistor: 0.220
+diag1_pin: !e_diag_pin
+driver_IHOLDDELAY: 8
+driver_TPOWERDOWN: 0
+driver_TBL: 2
+driver_TOFF: 3
+driver_HEND: 1
+driver_HSTRT: 5
+driver_PWM_FREQ: 2
+driver_PWM_GRAD: 4
+driver_PWM_AMPL: 240
+driver_PWM_AUTOSCALE: True
+driver_SGT: 3
+stealthchop_threshold: 1
```

Comment:
The driver and current settings should go in the stepper configs. Furthermore this should just be called tmc2130.cfg.

Furthermore:
1) Remove hold_current
2) Disable interpolation
3) Raise microsteps in steppers.cfg to 64 to make the steppers quieter after removing interpolation.

The above 3 changes makes the printer more accurate and less likely to cause artifacts due to runtime driver settings changes.

The extruder motor needs it's own stepper/ config too.

Reply:
Done everything execpt the microsteps change because the einsy is not able to work at 64 microsteps

Reply:
@jfestrada aah that's fair enough, forgot we are dealing with a 16mhz 8-bit controller here. Hmm. I kind of want to isolate those settings to the einsy boards so it's not hampering people if they decide to install a modern 32-bit board. Let me think about that!

Reply:
I was thinking about this also, and to my mind the best approach would be to rename the file so we could include the file steppers-einsy-board.cfg and steppers.cfg, the default should be the steppers-einsy-board.cfg. Take into account also that everything has been configured to work with the tmc2130. Also another thing to have in mind is that this is a "low cost" upgrade for the people that have a mk3s and want a little bit more, so they don't need to change the board at least that the board die.

Reply:
@jfestrada You're right about that last part. Since most of the steppers section kind of falls into "advanced usage", i guess it's OK for now to have the steppers.cfg use 32 microsteps. Let's leave it as it is 👍